configure.ac

```diff
@@ -229,16 +229,24 @@ AC_ARG_ENABLE([hardening],
 yes_no_try "$enable_hardening" "try"
 enable_hardening=$?
 
-dnl --enable-* options: features
+dnl --enable-* options: features within Pacemaker
 
+dnl @COMPAT This should be --with-systemd
 AC_ARG_ENABLE([systemd],
     [AS_HELP_STRING([--enable-systemd],
         [enable support for managing resources via systemd @<:@try@:>@])]
 )
 yes_no_try "$enable_systemd" "try"
 enable_systemd=$?
 
-dnl --enable-* options: features inherent to Pacemaker
+AC_ARG_ENABLE([deprecated-libs],
+    [AS_HELP_STRING([--enable-deprecated-libs],
+        [Install deprecated C libraries @<:@yes@:>@])]
```

Comment:
Build/Install ... maybe as a hint that if not enabled there is nothing left that would check the code in any way